src/App.js

```diff
@@ -1,13 +1,32 @@
+import React from 'react';
+import {
+  Routes,
+  Route,
+  Navigate,
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import Home from './routes/Home';
+import Calculator from './routes/Calculator';
+import Quotes from './routes/Quotes';
+import Layout from './components/Layout';
+import PageNotFound from './routes/PageNotFound';
+
 import './App.css';
-import Calculator from './components/Calculator';
-import Quotes from './components/Quotes';
 
 function App() {
   return (
-    <div className="App">
-      <Quotes />
-      <Calculator />
-    </div>
+    <>
+      <Router>
+        <Layout />
+        <Routes>
+          <Route path="/math-magicians" element={<Navigate to="/" />} />
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quotes" element={<Quotes />} />
+          <Route path="*" element={<PageNotFound />} />
+        </Routes>
```

Comment:
[OPTIONAL] Well done :clap: you did a lovely job :+1: but I suggested you change the Route code with my recommended code it will be cleaner your codebase :+1: check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
       <Routes>
          <Route path="/" element={<Layout />}>
            <Route index element={<Home />} />
            <Route path="/calculator" element={<Calculator />} />
            <Route path="/quotes" element={<Quotes />} />
            <Route path="*" element={<PageNotFound />} />
          </Route>
        </Routes>
```